tools/accuracy_checker/accuracy_checker/main.py

```diff
@@ -108,7 +108,15 @@ def build_arguments_parser():
     )
     parser.add_argument(
         '--tf_custom_op_config_dir',
-        help='path to directory with tensorflow custom operation configuration files for model optimizer',
+        help='path to directory with Model Optimizer transformation configuration files',
```

Comment:
Are you sure you want to edit this description? I think the old description is more apt.

Reply:
returned back